client-super-turtle-tracker/src/components/reusables/TurtleListItem.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import { useEffect, useState } from "react";
+import { StyleSheet } from "react-native";
+import { Avatar, List } from "react-native-paper";
+import { View } from ".";
+import { firebase } from "../../../env";
+import { capitalizeFirstLetter } from "../../utils/helpers/functions";
+import { Turtle } from "../../utils/interfaces/Turtle";
+
+const styles = StyleSheet.create({
+  item: {
+    borderBottomColor: "#cdc",
+    borderBottomWidth: 0.5,
+  },
+});
+
+export const TurtleListItem = ({ turtle }: { turtle: Turtle }): JSX.Element => {
+  // TODO: Store url in database if possible
+  const getPhoto = async (photoName: string): Promise<string> => {
+    const ref = firebase.storage().ref().child(`images/${photoName}`);
+    const url = await ref.getDownloadURL();
+    return url;
+  };
+
+  const [imageUrl, setImageUrl] = useState("");
+
+  useEffect(() => {
+    getPhoto(turtle.avatar).then((url) => {
+      setImageUrl(url);
+    });
+  }, [turtle.avatar]);
```

Comment:
This `useEffect` might need to be cleaned up because it is asynchronous, but I'll test it more in the future so don't worry about it here. It isn't giving me any warning currently.

Reply:
This will be fixed when #7 gets merged